command/version.go

```diff
@@ -19,6 +23,46 @@ func (c *VersionCommand) Name() string { return "version" }
 
 func (c *VersionCommand) Run(_ []string) int {
 	c.Ui.Output(c.Version.FullVersionNumber(true))
+
+	// Check the server version
+	client, err := c.Meta.Client()
+	if err != nil {
+		return 0
+	}
+	members, err := client.Agent().Members()
+	if err != nil {
+		return 0
+	}
+
+	// Return the latest version
+	latestServerRevision := ""
+	latestServerVersion, err := goversion.NewVersion("v0.0.0")
+	if err != nil {
+		return 0
+	}
+
+	for _, member := range members.Members {
+		if version, ok := member.Tags["build"]; ok {
+			semver, err := goversion.NewVersion(version)
+			if err != nil {
+				continue
+			}
+
+			if semver.GreaterThanOrEqual(latestServerVersion) {
+				latestServerVersion = semver
+				if revision, ok := member.Tags["revision"]; ok {
+					latestServerRevision = revision
+				} else {
+					latestServerRevision = "unknown"
+				}
+			}
+		}
+	}
```

Comment:
I'm having a little trouble with presenting the "highest revision" among all the servers as "the server version". Isn't that misleading in the case where there are mixed servers? And it'd be particularly confusing if you were making a CLI request to a client which didn't match that version, because the HTTP API could be missing features if the client was on an older version.

Or in other words, what does this really tell the operator that they wouldn't rather get from `nomad server members`?

Reply:
That kinda makes me think that we just shouldn't present it. Maybe we just provide the revision in the members output to let the operator know which servers are at which revision? I think that would make the most sense.

Reply:
Yeah, that sounds right to me too.